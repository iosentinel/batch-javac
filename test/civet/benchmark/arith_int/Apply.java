package benchmark.arith_int;

public class Apply {
  public static int a(Exp e, int[] env) {
    return e.eval(env);
  }
}
