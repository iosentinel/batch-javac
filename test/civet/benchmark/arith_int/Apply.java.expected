package benchmark.arith_int;

public class Apply {
    
    public Apply() {
        super();
    }
    
    public static int a(Exp e, int[] env) {
        return e.eval(env);
    }
    
    public static int a$1003(int[] env) {
        return benchmark.arith_int.Add.eval$1004(env);
    }
}